pkg/connector/ids.go

```diff
@@ -88,3 +92,55 @@ func (tc *TwitterClient) MakeEventSender(userID string) bridgev2.EventSender {
 		Sender:      MakeUserID(userID),
 	}
 }
+
+type MediaInfo struct {
+	UserID networkid.UserLoginID
+	URL    string
+}
+
+func MakeMediaID(userID networkid.UserLoginID, URL string) networkid.MediaID {
+	mediaID := []byte{1}
+	uID, err := strconv.ParseUint(ParseUserLoginID(userID), 10, 64)
+	if err != nil {
+		panic(err)
+	}
+	mediaID = binary.AppendUvarint(mediaID, uID)
+
+	bs := []byte(URL)
+	mediaID = binary.AppendUvarint(mediaID, uint64(len(bs)))
+	mediaID, err = binary.Append(mediaID, binary.BigEndian, bs)
+	if err != nil {
+		panic(err)
+	}
+
+	return mediaID
+}
+
+func ParseMediaID(mediaID networkid.MediaID) (*MediaInfo, error) {
+	buf := bytes.NewReader(mediaID)
+	version := make([]byte, 1)
+	_, err := io.ReadFull(buf, version)
+	if err != nil {
```

Comment:
The version should be validated (return an error if it's not the expected value of `1`)